.gitignore

```diff
@@ -41,7 +41,6 @@ unlinked.ds
 unlinked_spec.ds
 
 # Android related
-**/android/**/GeneratedPluginRegistrant.java
```

Comment:
Aren't these still generated code?  Generated code should be ignored.

Reply:
They are, this discussion said they need to be checked in, though: https://github.com/flutter/flutter/pull/13500#issuecomment-351080416